lib/rubocop/cop/ezcater/rspec_require_gql_error_helpers.rb

```diff
@@ -0,0 +1,35 @@
+module RuboCop
+  module Cop
+    module Ezcater
+      # Enforce use of GQLErrors helpers instead of throwing
+      # GraphQL::ExecutionErrors directly
+      #
+      # @example
+      #
+      #   # good
+      #   GQLErrors.summary_error("An error occurred")
+      #   GQLErrors.request_error("You can't access this", 401)
+      #   GQLErrors.field_error("is invalid", :first_name, "First Name")
+      #   GQLErrors.field_errors_for(my_model, context)
+      #   GQLErrors.field_errors_for(my_model, context, summary_error: "An error occurred")
+      #   GQLErrors.field_errors_for(my_model, context, field_mapping: { first: :first_name })
+      #
+      #   # bad
+      #   GraphQL::ExecutionError.new("An error occurred")
+      #   GraphQL::ExecutionError.new("You can't access this", options: { status_code: 401 })
+      class RspecRequireGqlErrorHelpers < Cop
```

Comment:
Just noticed that it probably does not make sense for this cop name to begin with "Rspec".

Reply:
Yeah - I've been mulling on this... wondering if the name is wrong in it's entirety actually. This cop really just disallows `GraphQL::ExecutionError` usage.... `PreventGraphQLExecutionErrorInvokation`?

Reply:
At some point, if necessary, we could introduce an option to only prevent object instantiation as well per our previous discussions.

Reply:
Just catching up from when I was out -- did we end up fixing the naming of this cop @tjwp @mjfaga ?

Reply:
@anishasmith On my list, haven't gotten to it yet. Will be done before I go on leave at the end of the week.

Reply:
PRs opened:
* https://github.com/ezcater/ezcater_rubocop/pull/30
* https://github.com/ezcater/ezcater_rubocop/pull/31